app/templates/views/broadcast/preview-areas.html

```diff
@@ -25,31 +25,43 @@
 
   {{ page_header("Choose where to send this alert") }}
 
-  {% for area in broadcast_message.areas %}
-    {% if loop.first %}
+  {% if broadcast_message.areas %}
+    <div>
+      <p class="govuk-!-margin-right-2 area-list">Area: </p>
       <ul class="area-list">
-    {% endif %}
-        <li class="area-list-item">
-          {{ area.name }}
-          <a class="area-list-item-remove govuk-button" data-module="govuk-button" role="button" href="{{ url_for('.remove_broadcast_area', service_id=current_service.id, broadcast_message_id=broadcast_message.id, area_slug=area.id) }}">
-            <svg id="area-list-item-remove__icon" width="11" height="11" viewbox="0 0 10 10" aria-hidden="true" focusable="false" xmlns="http://www.w3.org/2000/svg">
-              <title id="area__{{ loop.index }}">Remove {{ area.name }}</title>
-              <g transform="rotate(45),translate(1, -6)">
-                <line stroke-width="1.75" id="svg_1" y2="0" x2="6" y1="12" x1="6" stroke="currentColor" fill="none"/>
-                <line stroke-width="1.75" id="svg_3" y2="6" x2="0" y1="6" x1="12" stroke="currentColor" fill="none"/>
-              </g>
-            </svg>
-          </a>
-        </li>
-    {% if loop.last %}
+        {% for area in broadcast_message.areas %}
+          <li class="area-list-item">
+            {{ area.name }}
+            <a class="area-list-item-remove govuk-button" data-module="govuk-button" role="button" href="{{ url_for('.remove_broadcast_area', service_id=current_service.id, broadcast_message_id=broadcast_message.id, area_slug=area.id) }}">
+              <svg id="area-list-item-remove__icon" width="11" height="11" viewbox="0 0 10 10" aria-hidden="true" focusable="false" xmlns="http://www.w3.org/2000/svg">
+                <title id="area__{{ loop.index }}">Remove {{ area.name }}</title>
+                <g transform="rotate(45),translate(1, -6)">
+                  <line stroke-width="1.75" id="svg_1" y2="0" x2="6" y1="12" x1="6" stroke="currentColor" fill="none"/>
+                  <line stroke-width="1.75" id="svg_3" y2="6" x2="0" y1="6" x1="12" stroke="currentColor" fill="none"/>
+                </g>
+              </svg>
+            </a>
+          </li>
+          {% if loop.last %}
+            {{ govukButton({
+              "element": "a",
+              "text": "Add another area",
+              "href": url_for('.choose_broadcast_library', service_id=current_service.id, broadcast_message_id=broadcast_message.id),
+              "classes": "govuk-button--secondary govuk-!-margin-bottom-5"
+            }) }}
+          {% endif %}
```

Comment:
You don't need to do this.

Just put this below the `{% endfor %}`

```
        <li>
          {{ govukButton({
            "element": "a",
            "text": "Add another area",
            "href": url_for('.choose_broadcast_library', service_id=current_service.id, broadcast_message_id=broadcast_message.id),
            "classes": "govuk-button--secondary govuk-!-margin-bottom-5"
          }) }}
      </li>
```

Then remove the whole if